### ISSUE TITLE
There Are No Tests

### ISSUE THREAD

**Issue by lloy0076** [zoutapps/laravel-backpack-multiauth#2]
Subject says it all :)

**Comment by OliverZiegler** [comment]
I know. :)
This is the next point on the roadmap.
Help is highly appreciated ;)

**Comment by lloy0076** [comment]
@OliverZiegler - it seems my terseness probably said too much (or sounded like a criticism); in essence it's not but as I was mentioning to a colleague, "the sooner you start to write the tests, the better; and it's even better if you do the tests first."

:)

**Comment by OliverZiegler** [comment]
I totally agree! And I did not understand you as criticism. I wanted to write tests, but I'm currently not quite sure how to test my main functionality. Would be nice if you could leed me in the right directions with this :)

**Comment by tabacitu** [comment]
This is UNACCEPTABLE! How can this package have no unit tests? HOW?! When Backpack itself has [such a beautiful, complete, 100% test coverage and zero - absolutely none - just return true :-)](https://github.com/Laravel-Backpack/CRUD/blob/master/tests/BaseTest.php)? 

Haha... wouldn't worry about tests too much at the start. Though it would help with maintaining the package, I heard :-). A LOT. Can't wait to see that in Backpack ;-)

Sorry for this comment not being helpful. At all :-) Good job, keep up the good work and don't let haters like @lloy0076 let you down! :-) Kidding @lloy0076 of course.
